### PR TITLE
feat(levm): comparison opcodes

### LINKED ISSUE #468: Opcode: LT

### PR DESCRIPTION
**Pull request by juanimedone**
**Description**
Implements and unit tests the following comparison opcode operations:
- `LT`
- `GT`
- `SLT`
- `SGT`
- `EQ`
- `ISZERO`

Closes #468, #469, #470, #471, #472 and #473.

### CODE REVIEW

File: crates/levm/src/vm.rs
Comment (ilitteri) [inline crates/levm/src/vm.rs:1]: Let's use `left_hand_operand` or `lho` and `right_hand_operand` or `rho` for the non-commutative comparisons.

Comment (juanimedone) [reply crates/levm/src/vm.rs:1]: Done!

File: crates/levm/src/utils.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+use ethereum_types::U256;
+
+pub fn u256_to_i128(value: U256) -> i128 {
```
Comment (lazcanoluca) [inline crates/levm/src/utils.rs:3]: Maybe add a short comment explaining that this returns the signed integer from its 2's complement representation.

**Review by lazcanoluca — APPROVED** [review]
lgtm 🚀

File: crates/levm/src/utils.rs (hunk 1 above)
Comment (juanimedone) [reply crates/levm/src/utils.rs:3]: Sure, will do!

**Review by ilitteri — APPROVED** [review]
LGTM

File: crates/levm/src/utils.rs (hunk 1 above)
Comment (juanimedone) [reply crates/levm/src/utils.rs]: Casting to `i128` was incorrect in these cases so I removed that function.

**Review by lazcanoluca — APPROVED** [review]

**Pull request merged**